src/libcollections/btree/node.rs

```diff
@@ -1006,20 +926,17 @@ impl<'a, K, V> Handle<NodeRef<marker::Mut<'a>, K, V, marker::Internal>, marker::
                 }
             } else {
                 unsafe {
-                    Handle::new_edge(
-                        right.as_mut().cast_unchecked::<marker::Internal>(),
-                        self.idx - (B + 1)
-                    ).insert_fit(key, val, edge);
+                    Handle::new_edge(right.as_mut().cast_unchecked::<marker::Internal>(),
+                                     self.idx - (B + 1))
+                        .insert_fit(key, val, edge);
```

Comment:
I get that this is just indented once, but floating like this is, unconnected and unaligned to anything else, this `insert_fit` call looks weird to me.